baseapp/block_gas_test.go

```diff
@@ -125,7 +125,7 @@ func TestBaseApp_BlockGas(t *testing.T) {
 				require.Equal(t, []byte("ok"), okValue)
 			}
 			// check block gas is always consumed
-			baseGas := uint64(37352) // baseGas is the gas consumed before tx msg
+			baseGas := uint64(34824) // baseGas is the gas consumed before tx msg
```

Comment:
What does this change mean?

Reply:
This is a fix for the fact that the gas consumed by test varies depending on the implementation.
This is often arbitrarily changed by bump up etc.